App is crashing when navigating to Current Relation screen

**Describe the bug**

App is crashing when a user navigates to the Current Relation screen from the bottom navigation.

**To Reproduce**
Steps to reproduce the behavior:
1. Run application
2. Login user
3. Click the middle option on bottom navigation (3rd)
4. See App crashing error

**Expected behavior**

App should show the "Current Relation" screen

**Crash Log**

```
2019-03-20 20:40:24.611 12184-12184/org.systers.mentorship E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.systers.mentorship, PID: 12184
    android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class android.support.v7.widget.AppCompatTextView
    Caused by: android.view.InflateException: Binary XML file line #27: Error inflating class android.support.v7.widget.AppCompatTextView
    Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v7.widget.AppCompatTextView" on path: DexPathList[[zip file "/data/app/org.systers.mentorship-2q5LGYjCK5PYYslrFY4ezQ==/base.apk"],nativeLibraryDirectories=[/data/app/org.systers.mentorship-2q5LGYjCK5PYYslrFY4ezQ==/lib/arm64, /system/lib64, /vendor/lib64, /product/lib64]]
```

**Smartphone (please complete the following information):**
 - Device: Huawei
 - OS: Android
 - Version: 8 Oreo

**Additional context**

- This happens because of AndroidX migration.
- PR that caused this #134 
- Error occurs here: [app/src/main/res/layout/fragment_relation.xml](https://github.com/systers/mentorship-android/blob/develop/app/src/main/res/layout/fragment_relation.xml#L27)
- I think the solution is that `androidx.appcompat.widget.AppCompatTextView` class should be used instead.

--- a/app/src/main/res/layout/fragment_relation.xml
+++ b/app/src/main/res/layout/fragment_relation.xml
@@ -24,7 +24,7 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/tlMentorshipRelation" />
 
-    <android.support.v7.widget.AppCompatTextView
+    <androidx.appcompat.widget.AppCompatTextView
         android:id="@+id/tvNoCurrentRelation"
         android:layout_width="250dp"
         android:layout_height="wrap_content"

